apps/webapp/web/router.ex

```diff
@@ -0,0 +1,38 @@
+require Starwars.Router
+require Greeter.Router
+
+defmodule Webapp.Router do
+  use Webapp.Web, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_flash
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", Webapp do
+    pipe_through :browser # Use the default browser stack
+
+    get "/", PageController, :index
+  end
+
+  # act as final catch
+  scope "/starwars" do
+    forward "/", Starwars.Router
+  end
+
+  scope "/greeter" do
+    forward "/", Greeter.Router
```

Comment:
Can we say that **Webapp** is the "main app" ? ... Apparently the other apps are not accessible without this router, are they?